src/ENCRYPTO_utils/sndthread.cpp

```diff
@@ -62,6 +69,11 @@ void SndThread::add_snd_task(uint8_t channelid, uint64_t sndbytes, uint8_t* sndb
 
 }
 
+void SndThread::add_snd_task(uint8_t channelid, uint64_t sndbytes, uint8_t* sndbuf) {
+	//Call the method blocking but since callback is null nobody gets notified, other functionallity is equal
+	add_blocking_snd_task(NULL, channelid, sndbytes, sndbuf);
```

Comment:
I think we should change NULL to nullptr here (and at some other places in this PR)